src/expandable-section/expandable-section-header.tsx

```diff
@@ -50,7 +51,7 @@ interface ExpandableSectionHeaderProps extends Omit<ExpandableDefaultHeaderProps
   ariaLabelledBy?: string;
 }
 
-const ExpandableDefaultHeader = ({
+const ExpandableDeprecatedHeader = ({
```

Comment:
This is the function that returns the header when using the deprecated `header` prop —for all variants except navigation—, so this name is more appropriate.